common/djangoapps/custom_settings/views.py

```diff
@@ -69,3 +77,31 @@ def course_custom_settings(request, course_key_string):
 
         else:
             return HttpResponseNotAllowed("Bad Request", content_type="text/plain")
+
+
+def validate_course_open_date(settings, course_open_date):
```

Comment:
can we verify if any of the methods in common.djangoapps.util.date_utils might help here? (and any other place in this change)

Reply:
No there is not and this is specific to course cards and custom settings